spec/api-browser-window-spec.js

```diff
@@ -2525,6 +2525,20 @@ describe('BrowserWindow module', () => {
     })
   })
 
+  describe('window.getMediaSourceId()', () => {
+    it('returns valid source id', () => {
+      // Check format 'window:1234:0'.
+      const sourceId = w.getMediaSourceId()
+      const sourceIdSplit = sourceId.split(':')
+      expect(sourceIdSplit.length).to.equal(3)
+      expect(sourceIdSplit[0]).to.equal('window')
+      const windId = Number.parseInt(sourceIdSplit[1])
+      const contentId = Number.parseInt(sourceIdSplit[2])
+      expect(windId).to.be.a('number')
```

Comment:
Maybe rather:

```js
expect(sourceId).to.match(/^window:\d+:\d+$/)
```

since `parseInt('1aoeu')` => `1`

Reply:
Done, thx!

Reply:
I think the remaining failure is due to this change. Hi @nornagon any idea ?